src/garage/misc/tensor_utils.py

```diff
@@ -163,10 +241,32 @@ def split_tensor_dict_list(tensor_dict):
 
 
 def truncate_tensor_list(tensor_list, truncated_len):
+    """Truncate list of tensor.
+
+    Args:
+        tensor_list (list[numpy.ndarray]): List of tensors.
+        truncated_len (int): Length to truncate.
+
+    Returns:
+        numpy.ndarray: list of tensor.
+
+    """
     return tensor_list[:truncated_len]
```

Comment:
why wouldn't i just splice my list just like this?

Reply:
Yes, that is. For most of the functions, I didn't think about refactoring/replacing with the equivalent np function since this PR mainly addressed the errors when running the examples.